docs/src/examples/components/Dropdown/Types/DropdownExampleMultiple.shorthand.steps.ts

```diff
@@ -0,0 +1,31 @@
+import { Dropdown, DropdownSelectedItem } from '@stardust-ui/react'
+
+const selectors = {
+  triggerButton: `.${Dropdown.slotClassNames.triggerButton}`,
+  dropdownItem: (itemIndex = 1) =>
```

Comment:
may suggest to simplify the names, as those are not exported. The `dropdown` context is already delivered by file name, we could omit it for variable names. Maybe something like (just an option):

```
- triggerButton
- item
- removeItemIcon
```

Reply:
also, lets, please, make index an explicit arg (with no defaults) - it will make it easier to read the `steps` part of the tests

Reply:
applied both suggestions, thanks!